test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ObjectPath.java

```diff
@@ -148,4 +149,14 @@ private Object evaluate(String key, Object object, Stash stash) throws IOExcepti
 
         return list.toArray(new String[list.size()]);
     }
+
+    public XContentBuilder toXContentBuilder(XContent xContent) throws IOException {
```

Comment:
I see why I needed to put this method here. It needs access to the wrapped  `object` field in ObjectPath. I could make that accessible via a getter, but I think I like this better, not exposing the innards of ObjectPath.